addons/sale/wizard/sale_make_invoice_advance.py

```diff
@@ -80,7 +80,7 @@ def _prepare_invoice_values(self, order, name, amount, so_line):
             'partner_id': order.partner_invoice_id.id,
             'fiscal_position_id': (order.fiscal_position_id or order.fiscal_position_id._get_fiscal_position(order.partner_id)).id,
             'partner_shipping_id': order.partner_shipping_id.id,
-            'currency_id': order.pricelist_id.currency_id.id,
+            'currency_id': order.pricelist_id.currency_id.id or order.currency_id.id,  # TODO edm: needed? Seems working without? Shoudln't?
```

Comment:
```suggestion
            'currency_id': order.currency_id.id,
```

Same comment, I don't want to see `.pricelist_id.currency_id` in sale code anymore (except the compute method :D), the source of truth should always be the field :)